R/plotNhoods.R

```diff
@@ -741,3 +741,109 @@ plotNhoodMA <- function(da.res, alpha=0.05, null.mean=0){
 
   return(ma.p)
 }
+
+
+#' Plot the number of cells in a neighbourhood per sample and condition
+#'
+#' @param x A \code{\linkS4class{Milo}} object with a non-empty \code{nhoodCounts}
+#' slot.
+#' @param nhoods A logical, integer or character vector indicating the rows of \code{x} to use for
```

Comment:
It's not the rows of the Milo object - it's the rows of the `nhoods()` slot of the Milo object.